pkg/consts/consts.go

```diff
@@ -187,6 +187,16 @@ const (
 	BackoffJitterDefault = 1.0
 )
 
+// LB variables for dual-stack
+var (
+	// There's no "service.beta.kubernetes.io/azure-load-balancer" annotation before these 2 new ones are added.
+	// Service.Spec.LoadBalancerIP was used and it will be deprecated.
```

Comment:
Could you change the comments a little, e.g.

```
// Service.Spec.LoadBalancerIP has been deprecated and may be removed in a future release. Those two annotations are introduced as alternatives to set ipv4/ipv6 LoadBalancer IPs.
// Refer https://github.com/kubernetes/api/blob/3638040e4063e0f889c129220cd386497f328276/core/v1/types.go#L4459-L4468 for more details.
```

Reply:
Updated.